exe/etcd-env

```diff
@@ -40,10 +43,17 @@ def fetch(uri_str, limit = 10)
 end
 
 if __FILE__ == $PROGRAM_NAME
```

Comment:
@omarkhan what is the reason for this ? It fails when the gem is installed since __FILE__ is the full path.